packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -280,9 +281,30 @@ class EditableText extends StatefulWidget {
   /// The type of keyboard to use for editing the text.
   final TextInputType keyboardType;
 
+  /// The type of action button to use with the soft keyboard.
+  final TextInputAction textInputAction;
+
   /// Called when the text being edited changes.
   final ValueChanged<String> onChanged;
 
+  /// Called when the user submits editable content (ex: user presses the "done"
+  /// button on the keyboard).
+  ///
+  /// The default implementation of [onEditingComplete] executes 2 different
+  /// behaviors based on the situation:
+  ///
+  ///  - When a completion action is pressed, such as "done," "go," "send," or
+  ///    "search," the user's content is submitted to the [controller] and then
```

Comment:
`"search",`, with the comma outside the quotes, since the action isn't called `search,` with a comma.